docker-compose.yml

```diff
@@ -15,17 +15,6 @@ services:
         soft: -1
         hard: -1
 
-  # NOTE: This would be nice to develop with https:// locally, but it doesn't work, for whatever reason, so we need a system caddy instead
```

Comment:
This would never work "easily" (only if the deno piece was also running in docker instead of the host machine, and even still, there are other certificate issues), so I'm removing it.